includes/class-dispatcher.php

```diff
@@ -45,14 +52,20 @@ public static function fetch_forms( $username, $api_key = false ) {
 	public static function get_forms( $username = false ) {
 
 		$username = $username ? $username : Mad_Mimi_Settings_Controls::get_option( 'username' );
-		$api_key = Mad_Mimi_Settings_Controls::get_option( 'api-key' );
+		$api_key  = Mad_Mimi_Settings_Controls::get_option( 'api-key' );
 
 		if ( empty( $api_key ) ) {
+
 			return false;
+
 		}
 
-		if ( false === ( $data = get_transient( 'mimi-' . $username . '-lists' ) ) ) {
+		$data = get_transient( 'mimi-' . $username . '-lists' );
```

Comment:
@EvanHerman Do you want to use the same style as above? `"mimi-{$username}-lists"`